practiceRedditBot.py

```diff
@@ -4,13 +4,15 @@
 
 # import praw to use Reddit API
 import praw
+from botKeys import client_id, client_secret, password, user_agent, username

```

Comment:
```suggestion
from botKeys import *

```
makes it easier on eyes